src/llmq/quorums_commitment.h

```diff
@@ -79,6 +79,7 @@ class CFinalCommitment
             return false;
         }
         if (quorumPublicKey.IsValid() ||
+            !quorumVvecHash.IsNull() ||
```

Comment:
this seems bad... I don't really know enough about it though, so I'll just point it out and wait for you to say it's correct :)

Reply:
It's correct, but you made me question it for a second ;)
This method returns false if any of the fields is non-null. For the BLS stuff, that means if they are valid. For `uint256`, it means that it's not null (all-zero).